Added support for eZ Commerce Advanced on Platform.sh

vidarl opened the pull request:
Currently based on branch no_connection_to_solr_2.4, but should be rebased on https://github.com/ezsystems/ezcommerce-advanced/pull/3#pullrequestreview-204773981 once that is merged to 2.4

Edit: Jaroslaw has replaced https://github.com/ezsystems/ezcommerce-advanced/pull/3 with https://github.com/ezsystems/ezcommerce-advanced/pull/7... so needs to rebase on that one, once that one is merged

andrerom commented:
@vidarl / @fdege This is maybe something for the two of you to discuss
@vidarl Also see comment above.

reithor committed:
Add solr cores for "econtent"

1. add cores for econtent
2. uncomment solr config by default - as ezcommerce adv will not work whithout solr

vidarl committed:
[Platform.sh]Injecting ezcommerce SOLR config from environement variables

vidarl committed:
[Platform.sh]Added SOLR core config

vidarl committed:
[Platform.sh]Added eZ Commerce crons

andrerom reviewed (approved):
Rebased and pushed this so looks ok'ish now once we fix hard coding of install.
